cpp/src/gandiva/function_registry_string.cc

```diff
@@ -51,7 +51,7 @@ std::vector<NativeFunction> GetStringFunctionRegistry() {
 
       UNARY_OCTET_LEN_FN(octet_length, {}), UNARY_OCTET_LEN_FN(bit_length, {}),
 
-      UNARY_UNSAFE_NULL_IF_NULL(char_length, {}, utf8, int32),
+      UNARY_UNSAFE_NULL_IF_NULL(char_length, ({"character_length"}), utf8, int32),
```

Comment:
why the parenthesis?

Reply:
Hi Projjal,
I go continue this PR.
I really don't know why have this. i go remove!